midend/flattenInterfaceStructs.h

```diff
@@ -22,6 +22,12 @@ limitations under the License.
 
 namespace P4 {
 
+/*
```

Comment:
There is already a comment on the ReplaceStructs class which describes what it does. You should change that comment other than adding a new one here. This comment is also not very clear; it is not clear what is "another special struct".

Reply:
Will do.